chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -282,9 +283,11 @@
             cloud_available = true if name.include? "Cloud"
           end
 
+          autoyast_template="autoyast"
```

Comment:
Style/SpaceAroundOperators: Surrounding space missing for operator `=`. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)